controls/V-72253.rb

```diff
@@ -57,20 +57,12 @@
 
 The SSH service must be restarted for changes to take effect."
 
-  # Must be either or both of these ciphers
-  # @todo - replace with be_in when available
-  describe.one do
-    describe sshd_config do
-      its('MACs') { should match /^hmac-sha2-256,hmac-sha2-512$/ }
-    end
-    describe sshd_config do
-      its('MACs') { should match /^hmac-sha2-256$/ }
-    end
-    describe sshd_config do
-      its('MACs') { should match /^hmac-sha2-512$/ }
-    end
-    describe sshd_config do
-      its('MACs') { should match /^hmac-sha2-512,hmac-sha2-256$/ }
+  @macs = inspec.sshd_config.params("macs")
+  if !@macs.nil?
+    @macs.first.split(",").each do |mac|
+      describe mac do
+        it { should be_in ['hmac-sha2-256', 'hmac-sha2-512'] }
```

Comment:
I'm not 100% sure that sha-512 works in FIPS mode (and I'm 80% sure that it doesn't in all versions of RHEL). This should be checked before it's pushed.

We're going to have to toggle some of these based on FIPS being enabled on the target system.